site/docs/src/json/tenants/responses.json

```diff
@@ -30,9 +39,9 @@
         },
         "security": "TLS",
         "setPasswordEmailTemplateId": "a9aba13e-0125-4fd7-a2b1-aaa146b02423",
-        "unverified" : {
-          "allowEmailChangeWhenGated" : false,
-          "behavior" : "Allow"
```

Comment:
Thanks for the cleanup, probably would have been better done in a sep PR (that wouldn't need any review :) ).